### PR TITLE
Functional flashcards

### LINKED ISSUE #614: Update Flash Cards Project
# Project to be updated: [Flash Cards](https://frontend.turing.edu/projects/flash-cards.html)

## Summary of changes needed: 
- Currently, flash cards both describes and demands the use of classes
- The project needs to be restructured to use functions and object literals where necessary
- The project spec should give students a reasonable path to follow in laying out project architecture, roughly following an MVC pattern.
- All group projects should make clear that a failure to participate can result in an individual failing the project, even if the project is otherwise passing
- Update rubrics to use the Yes, Not Yet, Wow scale that was piloted on M1 projects.  Use M1 projects as a guide for consistency.

### PR DESCRIPTION
**Pull request by robbiejaeger**
This is an updated project spec that converts all of the classes and methods to be functions. The project spec follows [this implementation](https://github.com/turingschool-examples/flashcards-starter/pull/759).

Closes #614.

### CODE REVIEW

File: projects/module-2/flash-cards.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+round.currentCard;    // => { id: 12,
+            	        //      question: 'What is Travis\'s favorite stress reliever?',
+            	        //      answers: ['listening to music', 'watching Netflix', 'playing with bubble wrap'],
+            	        //      correctAnswer: 'playing with bubble wrap'
+            	        //    }
+
+calculatePercentCorrect(round); // => 50
+```
+</section>
```
Comment (kaylagordon) [inline projects/module-2/flash-cards.md:155]: the examples provided in each iteration are REALLY clear.

File: projects/module-2/flash-cards.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+***Cards***
+
+- A `card` represents a single flashcard
+- Each card has an id, a question, possible answers, and a correct answer
+
+<section class="answer">
+### For Example  
+
+```js
+const card = createCard(1, 'What is Robbie\'s favorite animal', ['sea otter', 'pug', 'capybara'], 'sea otter');
```
Comment (kaylagordon) [inline projects/module-2/flash-cards.md]: Should you include the expected output?

File: projects/module-2/flash-cards.md [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+
+For the rubric sections below, you will be scored as **Wow**, **Yes** or **Not Yet** depending on whether you have demonstrated competency in that area. Each section lists examples of what types of things we may be looking for as demonstrations of competency. Just as there are many ways to approach code, there are many many ways to demonstate competency.  There are just some examples. 
+
+### Functional Expectations
+* Wow: Application fulfills all requirements *as well as* an extension from iteration 4.
+* Yes: Application fulfills all requirements of iterations 1-3 without bugs.
+* Not Yet: Application crashes (game is not playable) or has missing functionality or bugs.
+
+### JavaScript & Style / Functional Programming
+- Code is divided into logical components each with a clean, single responsibility
```
Comment (kaylagordon) [inline projects/module-2/flash-cards.md:233]: In m1, we've added something like "On track looks like..." before the bullets for clarity in each rubric section

File: projects/module-2/flash-cards.md [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+
+## Day-One Deliverable
+- By end of day on kick off day: Submit your GH Repo [here](https://docs.google.com/forms/d/e/1FAIpQLScsgrJD22g9WnUj7-3gXMHFSPqkk9rTt86kbRTEDGfGCIMLVA/viewform?usp=sf_link)
+
+## Testing
+
+You should be using the prompts below to develop tests that will drive your implementation of code.
+
+Your testing suite should test all of the functionality of the game that you have implemented. Remember to test all possible outcomes (happy/sad/etc). Ask yourself:  
+  - What does the function return?
```
Comment (kaylagordon) [inline projects/module-2/flash-cards.md:41]: These guiding questions are really helpful

**Review by kaylagordon — COMMENTED** [review]
Looks great!

File: projects/module-2/flash-cards.md (hunk 3 above)
Comment (robbiejaeger) [reply projects/module-2/flash-cards.md:233]: Fixed!

File: projects/module-2/flash-cards.md (hunk 2 above)
Comment (robbiejaeger) [reply projects/module-2/flash-cards.md]: Fixed!

**Pull request merged**